src/webportal/src/app/user/fabric/user-profile/ssh-list.jsx

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) Microsoft Corporation
```

Comment:
Please use this header:

```javascript
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

```

Reply:
Will modifiy it. And looks like that still have many files write that.

Reply:
Yes, they are the old headers. We haven't modified them. This one is the new header.

Reply:
ok